.travis/k3s-install.sh

```diff
@@ -35,30 +35,42 @@ if ! [[ $? ]] ; then
   exit 1
 fi
 
+if [ -x /usr/local/bin/kubectl ]; then
+  KUBECTL=/usr/local/bin/kubectl
+elif which kubectl > /dev/null; then
```

Comment:
Some lightweight VMs (or cloud instances) do not have `which` installed. Use `command -v` instead.

Reply:
I didn't know about that